indicator-nara/01-inleiding.Rmd

```diff
@@ -0,0 +1,109 @@
+# Inleiding {#h1:inleiding}
+Driekwart van de voor Vlaanderen en Europa meest waardevolle natuurtypes, steunen voor hun voortbestaan op grondwater. Grondwaterafhankelijke vegetaties komen voor in en op de rand van kwelgebieden, beekvalleien, vijvers en poelen. 
+De vochtige gebieden vertonen vaak veel gradiënten en herbergen daarom een hoge biodiversiteit (De Becker, 2020). 
+Als het grondwater door verdroging van samenstelling verandert, of systematisch buiten het bereik van de plantenwortels en de bovengrondse delen van waterplanten valt, krijgen typische soorten het moeilijk. 
+Slagen die soorten er niet in om zich aan te passen of om hun leefgebied te verschuiven, dan dreigen ze te verdwijnen. 
+Dat risico wordt groter als bijvoorbeeld het landgebruik of het grondwaterregime in de omgeving de beschikbare ruimte om uit te wijken beperkt. 
+
+# Monitoring droogte in Vlaanderen {#h2:monitoring}
+Monitoring is het op regelmatige tijdstippen gestandaardiseerd beschrijven van een maatstaf. 
+Deze maatstaf moet toelaten om een bepaalde doelstelling of toestand te beoordelen.
+Toegepast op dit onderzoek, wil dit zeggen dat we een droogte-indicator zoeken, waarmee de droogte op een gestandaardiseerde wijze in droogtegevoelige locaties/vegetaties in Vlaanderen op jaarbasis kan opgevolgd worden.
+
+Een gestandaardiseerde wijze wordt in dit onderzoek beschouwd als een vast aantal meetpunten waar het grondwaterpeil op reguliere basis op een uniforme manier wordt gemeten.
+Het vast aantal meetpunten krijgt vorm door middel van een meetnet (zie \@ref(h2-1:meetnet)).
+Om droogte adequaat en zo precies mogelijk te kunnen opvolgen, menen we dat dagelijkse metingen aangewezen zullen zijn.
+Als maatstaf gaan we op zoek naar een gepaste droogte-indicator (zie \@ref(h2-2:indicator)).
+
+## Droogtemeetnet {#h2-1:meetnet}
+Om in Vlaanderen een globaal beeld te krijgen van de evolutie van de grondwaterstanden op middellange tot lange termijn in verdrogingsgevoelige vegetaties werd een meetnet van een honderdtal punten ontworpen. 
+Op schaal Vlaanderen is dit een relatief laag aantal. 
+Wil men op deze schaal veralgemeenbare uitspraken kunnen maken, moet de keuze van deze punten zo goed mogelijk ruimtelijk gebalanceerd gebeuren. 
```

Comment:
Je hebt voor het grootste deel gelijk. Een ruimtelijke balancering gebaseerd op een vaste set punten, kan nog steeds ernstig gebiased zijn (bijv. als die set niet representatief is voor de variabele die je wenst te onderzoeken).
Ik vind wel een ruimtelijke balancering een goede techniek om representatief te kunnen zijn. Volgens mij is het een nodige voorwaarde, maar daarom nog niet een voldoende voorwaarde. In die zin zal ik de zin nog nuanceren.
Stellen dat een representatief staal enkel kan met een aselecte trekking, vind ik niet helemaal juist. Een aselecte trekking kan per toeval ook totaal 'verkeerd' (gebiased) zijn. Denk ik toch, kan me vergissen

Voor het meetnet hebben we niet uitsluitend beroep gedaan op bestaande punten. We zijn uitgegaan van de dekking aan grondwatertypen in een hok. Bij het selecteren van percelen in dat hok hebben we wel voorrang gegeven aan bestaande meetpunten, dus daar kan zeker een bias op zitten. Soms kwamen er geen meetpunten in voor en moet er voor een nieuw punt gezorgd worden. Hier heb ik wel nog een voorwaarde aan gekoppeld, nl. dat het in een perceel met natuurbeheer moet liggen (maakt administratief makkelijker en ook wel meer garantie op duurzaamheid).

Reply:
Wanneer we spreken over unbiased, dan hebben we het over de _verwachte waarde_, niet over een individuele steekproef. Geen enkele individuele steekproef zal perfect unbiased zijn. Enkel aselecte (of systematische) steekproeven garanderen een onvertekende verwachte waarde.

Als je de som van 5 dobbelstenen als variabele neemt, dan is de verwachte waarde 17.5. Iets wat je in de praktijk nooit zal tegenkomen. Als je die steekproeftrekking vaak herhaald en daarvan het gemiddelde krijgt zal je bij faire dobbelstenen wel aan de verwachte waarde van 17.5 komen.

Van zodra je afwijkt van een aselecte of systematische steekproef, heb je niet langer de garantie van een onvertekende _verwachte waarde_.

Reply:
Ok, een aselecte proef zou in het geval van een meetnet betekenen, dat je elk jaar opnieuw een trekking doet. Of vergis ik me ?
Dat is natuurlijk in deze concrete situatie niet echt doenbaar ;-) Maar het is wel heel goed om te weten hoe het eigenlijk best zou gebeuren.

Reply:
Neen. Je kan perfect een aselect (of systematische) steekproef perfect meerdere jaren gebruiken. Je moet enkel zorgen dat je de steekproef kan "verversen" wanneer de steekproefpopulatie wijzigt.

Voor de meeste Natura 2000 meetnetten gebruiken we een aselecte steekproef waarbij we locaties jaarlijks of driejaarlijks opvolgen.

Reply:
OK,  ik was verkeerd door te zeggen dat door een aselecte trekking je uitkomst kan gebiased zijn.  Per definitie is die steeds ungebiased. Je resultaat kan er wel nog serieus naast zitten, de kans wordt hierop wel kleiner naarmate je steekproef groter wordt. Het is de variabiliteit van die variabele die dan richtinggevend is om te bepalen hoe groot je trekking moet zijn. 
Het ruimtelijk balanceren is een hulpmiddel om die variantie wat te verkleinen (of zoals Floris schreef: de efficiëntie vergroot), net zoals door het stratifiëren op de variabelen die ook een mogelijke impact hierop hebben.
Klopt die zienswijze ?

Reply:
Het is zoals Thierry aangeeft: 1 steekproefuitkomst (en in de praktijk heb je er maar één) wijkt vaak af van de verwachte waarde van de schatter (die voor een onvertekende schatter gelijk is aan de populatieparameter). Belangrijk is dat je wel het betrouwbaarheidsinterval van die ene uitkomst nog steeds kunt interpreteren in relatie tot de verwachte waarde (met bv. 95% betrouwbaarheid). Een aselecte steekproefstrategie heb je nodig om te kunnen stellen dat de verwachte waarde van de schatter gelijk is aan de populatieparameter, waarin we zijn geïnteresseerd.

De keuze van welke aselecte steekproefstrategie je gebruikt heeft invloed op de distributie van de schatter (distributie volgens herhaalde ficitieve trekkingen), en dus op de breedte van bovenvermeld betrouwbaarheidsinterval (in de veronderstelling dat je dit voordeel benut in de berekening van varianties). Steekproefstrategie heeft dus sowieso invloed op de afwijking van de steekproefuitkomst tov de echte waarde. Een schatter volgens GRTS is efficiënter dan volgens simple random sampling: dwz zijn distributie is smaller.

> een aselecte proef zou in het geval van een meetnet betekenen, dat je elk jaar opnieuw een trekking doet

In principe is de spatiale doelpopulatie (verspreiding van de locaties van interesse) continu in verandering, dus idealiter heb je continue updates van het volledige steekproefkader. Dan zou je in theorie jaarlijks (maar in praktijk bv. zesjaarlijks) opnieuw een trekking doen om de steekproefstrategie te blijven respecteren. Een GRTS-master heeft het gigantische voordeel dat het merendeel van de steekproefelementen tussen die trekkingen gemeenschappelijk zullen zijn. Dus ja, opnieuw trekken is in principe de bedoeling, in de praktijk komt dit neer op beperkte wijzigingen van de steekproef. Dit is trouwens in overeenstemming met het antwoord van Thierry. Uiteindelijk hangt veel af van de snelheid waarmee de doelpopulatie wijzigt, om te kunnen oordelen welke termijn nog redelijk is om een steekproef vast te houden. Een perfect steekproefkader hebben we overigens quasi nooit, dus daar komt al een ongekende vertekening insluipen. Het zo goed mogelijk krijgen of houden van het steekproefkader is dan ook geen overbodige luxe om een zinvolle steekproef te kunnen nemen.

NB. het woord 'representatief' heeft allerlei betekenissen en wordt beter vermeden voor statistische context. Zie bv.:

- Kruskal W. & Mosteller F. (1979). Representative Sampling, III: The Current Statistical Literature. International Statistical Review / Revue Internationale de Statistique 47 (3): 245–265. https://doi.org/10.2307/1402647.
  - _Citation on p. 259: "Representative sampling is a term easier to avoid because it is disappearing from the technical vocabulary. At different times it has been used for random sampling, proportionate sampling, quota sampling, and purposive sampling. In general, if often denotes the aims of representing a population well with a sample"_
- Kruskal W. & Mosteller F. (1980). Representative Sampling, IV: The History of the Concept in Statistics, 1895-1939. International Statistical Review / Revue Internationale de Statistique 48 (2): 169–195. https://doi.org/10.2307/1403151.
- p. 186 in: Quataert P. (2011). Methodological and statistical aspects of indices of biotic integrity to assess the ecological condition of surface waters. PhD theses of the Research Institute for Nature and Forest, Nr. INBO.T.2011.1. Instituut voor Natuur- en Bosonderzoek (INBO), Brussel, 218 p.
- "Over de betekenissen van ‘representatief’ in de KRW-literatuur": https://drive.google.com/file/d/1KF3MNZ0P93hkMcEakzl-eXxEop_6i4Ta/view?usp=sharing

Reply:
Floris, bedankt voor de toelichting! Ik denk dat ik het begrijp.
Ivm representatief: jammer dat het woord een statistische connotatie heeft. Mag ik 'weerspiegeling' gebruiken of hoe moet ik anders zeggen dat het streefdoel is dat we met een steekproef (zo unbiased mogelijk) iets over de gehele populatie willen zeggen.

Reply:
OK. 'We pogen zo goed mogelijk te weerspiegelen' is ook een van de vele mogelijke manieren om die gedachte te beschrijven. Maar belangrijker is dat de eigenschappen van de methode beschreven zijn - dat zijn ze via het achterliggend rapport.